client/app/pages/admin/Jobs.jsx

```diff
@@ -80,6 +80,8 @@ class Jobs extends React.Component {
     const { isLoading, error, queueCounters, startedJobs, overallCounters, workers, activeTab } = this.state;
 
     const changeTab = (newTab) => {
+      $location.hash(newTab);
+      $rootScope.$applyAsync();
       location.replace(`${location.pathname}#${newTab}`);
```

Comment:
This line should be removed, no? :slightly_smiling_face:

Reply:
Which line? 😉